src/BenchmarkDotNet.Core/Environments/HostEnvironmentInfo.cs

```diff
@@ -85,7 +91,7 @@ protected HostEnvironmentInfo()
         public override IEnumerable<string> ToFormattedString()
         {
             yield return $"{BenchmarkDotNetCaption}=v{BenchmarkDotNetVersion}, OS={OsVersion.Value}";
-            yield return $"Processor={ProcessorName.Value}, ProcessorCount={ProcessorCount}";
+            yield return $"Processor={ProcessorName.Value}, LogicalCoresCount={LogicalCoresCount}, PhysicalCoresCount={PhysicalCoresCount.Value?.ToString() ?? RuntimeInformation.Unknown}, PhysicalProcessorsCount={PhysicalProcessorsCount.Value?.ToString() ?? RuntimeInformation.Unknown}";
```

Comment:
This line is huge. Could we introduce additional variables? (Especially for things like `PhysicalCoresCount.Value?.ToString() ?? RuntimeInformation.Unknown`, it's not a good idea to use complicated expression inside very long interpolation string).